### PR TITLE
Add lmdbxx package

### PR DESCRIPTION
**Pull request by isaachier**

### CODE REVIEW

File: cmake/projects/lmdbxx/hunter.cmake [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+  PACKAGE_NAME lmdbxx
+  VERSION "0.9.14.0"
```
Comment (ruslo) [inline cmake/projects/lmdbxx/hunter.cmake]: Set it as default.

File: examples/lmdbxx/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+include("../common.cmake")
+
+project(download-lmdbxx)
+
+hunter_add_package(lmdbxx)
+
+find_package(lmdbxx CONFIG REQUIRED)
+
+set(CMAKE_CXX_STANDARD 11)
```
Comment (ruslo) [inline examples/lmdbxx/CMakeLists.txt]: Please remove, see:
* http://cgold.readthedocs.io/en/latest/tutorials/toolchain/globals/cxx-standard.html#summary

File: cmake/projects/lmdbxx/hunter.cmake (hunk 1 above)
Comment (isaachier) [reply cmake/projects/lmdbxx/hunter.cmake]: Sorry messed this up after the lmdb change.

File: examples/lmdbxx/CMakeLists.txt (hunk 2 above)
Comment (isaachier) [reply examples/lmdbxx/CMakeLists.txt]: I think this is going to affect the working builds. Will make another pull request to https://github.com/ingenue/hunter for pkg.lmdbxx after my build completes.

Comment (isaachier) [reply examples/lmdbxx/CMakeLists.txt]: https://github.com/ingenue/hunter/pull/185

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2962
* https://travis-ci.org/ingenue/hunter/builds/313617104?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Uploading:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2966
* https://travis-ci.org/ingenue/hunter/builds/313722622?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.19.188

**Pull request merged**